src/Mvc/Mvc.Testing/src/Handlers/RedirectHandler.cs

```diff
@@ -173,6 +173,7 @@ private static async Task<(Stream originalBody, Stream copy)> CopyBody(HttpReque
         private bool IsRedirect(HttpResponseMessage response) =>
             response.StatusCode == HttpStatusCode.MovedPermanently ||
                 response.StatusCode == HttpStatusCode.Redirect ||
+                response.StatusCode == HttpStatusCode.RedirectMethod ||
                 response.StatusCode == HttpStatusCode.RedirectKeepVerb ||
                 (int)response.StatusCode == 308;
```

Comment:
I know this isnt your code, but out of curiosity, why isnt this an `enum` as well (308)?

Reply:
My own reason was to create PR complying the recommendation "your PR is small, focused, and avoids making unrelated changes" - as this PR was my very first experience with Git, GitHub, aspnetcore and collaborating at all. ;-)
And I was afraid of the reviewers. :-)

Reply:
There's no enum value for 308 (at least on net4xx when this was written). It might be possible to update it to use the enum value, but it doesn't add value for us to care and update it.